styles/pages/impact.module.scss

```diff
@@ -0,0 +1,119 @@
+@import '../variables.scss';
+
+.container {
+  margin: 0 auto;
+  max-width: 50rem;
+  padding: 0 1em;
+}
+
+.pronouns {
+  margin-top: 0;
+  padding-top: 0;
+}
+
+.name {
+  margin-bottom: 0;
+  padding-bottom: 0;
+}
+
+.text-center-holographic {
+  z-index: -100;
+  position: absolute;
+  left: 626.37px;
+  top: 605.3px;
+  padding-left: 3px;
+  text-align: center;
+  font-family: $font-header;
+  background-image: linear-gradient(rgba(255, 0, 92, 0.3996),
+  rgba(177, 100, 255, 0.54),
+  rgba(51, 255, 206, 0.54),
+  rgba(125, 255, 46, 0.54));
+  background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
```

Comment:
Hm, I'm not able to get this to work as intended on chrome:
![Kapture 2021-09-03 at 22 07 59](https://user-images.githubusercontent.com/14893287/132083244-898a85ea-cae4-459f-825d-794a5eb34379.gif)